lib/js/components/ProgressDonutChart/ProgressDonutChart.vue

```diff
@@ -0,0 +1,287 @@
+
+$progress-donut-chart-width: 40px;
+$progress-donut-chart-height: 40px;
+$progress-donut-chart-circle-stroke-width: 4;
+$pi: 3.1416;
+$progress-donut-chart-circle-radius: 18; // (width / 2) - (stroke-width / 2)
+$circle-circumference: 2 * $pi * $progress-donut-chart-circle-radius;
+$progress-donut-chart-loading-dot-width: 2px;
+$progress-donut-chart-loading-dot-height: 2px;
+
+$progress-donut-chart-range-colors: (
+	'primaryMedium': $color-primary-data-medium,
+	'primary': $color-primary-data,
+	'primaryWeak': $color-primary-data-weak,
+	'primaryGhost': $color-primary-data-ghost,
+	'neutralMedium': $color-neutral-data-medium,
+	'neutral': $color-neutral-data,
+	'neutralWeak': $color-neutral-data-weak,
+	'neutralGhost': $color-neutral-data-ghost,
+	'infoMedium': $color-info-data-medium,
+	'info': $color-info-data,
+	'infoWeak': $color-info-data-weak,
+	'infoGhost': $color-info-data-ghost,
+	'successMedium': $color-success-data-medium,
+	'success': $color-success-data,
+	'successWeak': $color-success-data-weak,
+	'successGhost': $color-success-data-ghost,
+	'warningMedium': $color-warning-data-medium,
+	'warning': $color-warning-data,
+	'warningWeak': $color-warning-data-weak,
+	'warningGhost': $color-warning-data-ghost,
+	'failMedium': $color-fail-data-medium,
+	'fail': $color-fail-data,
+	'failWeak': $color-fail-data-weak,
+	'failGhost': $color-fail-data-ghost,
+);
+
+@keyframes dot-flashing {
+	0% {
+		background-color: $color-neutral-text;
+	}
+
+	50%,
+	100% {
+		background-color: transparent;
+	}
+}
+
+.progressDonutChart {
+	position: relative;
+	width: $progress-donut-chart-width;
+	height: $progress-donut-chart-height;
+	padding: 0;
+	margin: 0;
+	overflow: hidden;
+	box-sizing: border-box;
+
+	&__svg {
+		display: block;
+		width: $progress-donut-chart-width;
+		height: $progress-donut-chart-height;
+	}
+
+	&__circle {
+		fill: none;
+		stroke-width: $progress-donut-chart-circle-stroke-width;
+		// Disabled for v3.0
+		//stroke-linecap: round;
+		//stroke-linejoin: round;
+		transition: all $default-transition-time ease-out;
+		transform: rotate(90deg);
+		transform-origin: 50% 50%;
+	}
+
+	&__thumb {
+		stroke: $color-neutral-data-ghost;
+	}
+
+	&__track {
+		@each $class, $color-name in $progress-donut-chart-range-colors {
+			&.-#{$class} {
+				stroke: $color-name;
+			}
+		}
+
+		stroke-dasharray: $circle-circumference;
+		stroke-dashoffset: calc(
+			#{$circle-circumference} - (#{$circle-circumference} * (var(--length, 0) / 100))
+		);
+	}
+
+	&__loader {
+		stroke-dasharray: $circle-circumference;
+		stroke: $color-neutral-background-strong;
+		stroke-dashoffset: #{$circle-circumference - ($circle-circumference * (25 / 100))};
+		animation: a-spinAround 2s infinite linear;
+		transform: rotate(0deg);
+	}
+
+	&__label {
+		@include info-m-default-bold;
+
+		display: flex;
+		position: absolute;
+		top: 0;
+		left: 0;
+		width: 100%;
+		height: 100%;
+		justify-content: center;
+		flex-wrap: nowrap;
+		align-items: center;
```

Comment:
alphabetical order?

Reply:
@kvas-damian enable https://github.com/hudochenkov/stylelint-order/blob/master/rules/properties-alphabetical-order/README.md if you think it's valuable 🙂

Reply:
stop spying us. You have a lot to do in other ticket :trollface:

Reply:
maybe we should enable https://github.com/hudochenkov/stylelint-order/blob/master/rules/properties-alphabetical-order/README.md ?

Reply:
@rogatty https://github.com/bethinkpl/design-system/pull/276